indico/modules/core/client/js/session_bar.js

```diff
@@ -80,6 +71,7 @@
       },
       events: {
         show() {
+          // eslint-disable-next-line no-undef
```

Comment:
You can avoid both `eslint-disable-next-line` statements by importing lodash properly ;) `import _ from 'lodash';`

Reply:
Hm. So why wasn't it imported to begin with? 🤔

Reply:
Anyway, I wouldn't dwell on this cause it's going away once the pull-down menu hits the PR backlog.

Reply:
bad old legacy times ;)